library/src/main/java/se/emilsjolander/stickylistheaders/StickyListHeadersAdapter.java

```diff
@@ -0,0 +1,147 @@
+package se.emilsjolander.stickylistheaders;
```

Comment:
Add copyright details to all the new files, if old files then retain original library copyright